hardware/BleBox.cpp

```diff
@@ -586,14 +569,9 @@ namespace http {
 			}
 
 			std::string hwid = request::findValue(&req, "idx");
-			if (hwid == "")
-				return;
-			int iHardwareID = atoi(hwid.c_str());
-			CDomoticzHardwareBase *pBaseHardware = m_mainworker.GetHardware(iHardwareID);
+			CDomoticzHardwareBase *pBaseHardware = GetHardwareBase(hwid, HTYPE_BleBox);
```

Comment:
On first version i added this method to mainworker, but later i saw that all occurence exists in webserver class.
But ok, i will add new method to mainworker.